app/code/community/Elgentos/CodebaseExceptions/Helper/Data.php

```diff
@@ -41,4 +41,35 @@ public function insertException($reportData) {
 
         $this->client->notifyOnError($reportData[0],$backtraces);
     }
+
+    /**
+     * @param string $message
+     */
+    public function insertLog($message) {
+        $messageArray = explode("\n", $message);
+        if (count($message) < 1) {
+            exit;
+        }
+        $header = '';
+        //reformatting exception
+        if (isset($messageArray[0])) {
+            $header .= $messageArray[0];
+            unset($messageArray[0]);
+        }
+        if ($header == '' && isset($messageArray[1])) {
+            $header .= $messageArray[1];
+            unset($messageArray[1]);
+        }
+        foreach($messageArray as $key => $line) {
+            $messageArray[$key] = array('file' => $line);
+        }
+        $notice = new \Airbrake\Notice;
+        $notice->load(array(
+            'errorClass'   => 'PHP Error',
+            'backtrace'    => $messageArray,
+            'errorMessage' => $header,
```

Comment:
Could you please stick to the naming that we already have? That is, `$header` becomes `$errorMessage` and `$messageArray` becomes `$backtrace` (or `$backtraceArray`).